homeassistant/components/music_assistant/media_player.py

```diff
@@ -397,6 +447,14 @@ async def _async_handle_play_media(
             elif await asyncio.to_thread(os.path.isfile, media_id_str):
                 media_uris.append(media_id_str)
                 continue
+            # last resort: search for media item by name/search
+            if item := await self.mass.music.get_item_by_name(
+                name=media_id_str,
+                artist=artist,
+                album=album,
+                media_type=MediaType(media_type) if media_type else None,
+            ):
+                media_uris.append(item.uri)
```

Comment:
Just as general note: if we add this, we probably will have a hard time removing this again. So if we want users to implement their own search, we should do that now, instead of adding this.

Reply:
This feature is actually used A LOT by MA users so I dont want to remove it.
Yes, at some point we will have a search feature from within media browsing but without the ability to play media by name, users will be very dissapointed.